00-settings/settings_base.tex

```diff
@@ -188,10 +189,14 @@
 \setlength{\footskip}{1.5cm}
 \addtolength{\textheight}{-.5cm}
 
+% Set numbering
+\setcounter{secnumdepth}{3}
```

Comment:
This is to get the subsubsections in the main TOC? Is it specific to Overleaf compat? Does it cause issues there?

The idea was to have a general main TOC then use the chapter mini-TOCs for deeply nested titles but I'm open to change it.

Reply:
No, for toc depth it's tocdepth, this one is just to make sure the numbers are shown on subsubsections, which was not happening i think